### DISCUSSION TITLE
How to use new system with lagrangian formulation in dynamic analysis?

### DISCUSSION THREAD

**Discussion by xiaoruiqi** [idaholab/moose#23169] [Q&A General] [answered]
Hi, every one! 
I met an error when I was doing dynamic analysis by setting `new_system = true` in `DynamicTensorMechanics` kernel. The error message is as follows:
```
*** ERROR ***
 The following error occurred in the object "MOOSE Problem", of type "FEProblem".
  Material property 'stress', requested by 'TM_DynamicTensorMechanics0' is not defined on block plate
  Material property 'Jacobian_mult', requested by 'TM_DynamicTensorMechanics0' is not defined on block plate
```
However, the name of Cauchy stress and its jacobian should be "cauchy_stress" and "cauchy_jacobian"  for lagrangian formulation. Is it a bug and we just need to change the name from "cauchy_stress" and "cauchy_jacobian" to "stress" and "Jacobian_mult"?

**Comment by GiudGiud** [comment]
Hello

the others will know more but I think you just need both the object that defines and the ones that consume this stress material property to be using the new system 

Guillaume

**Comment by hugary1995** [comment, marked as answer]
For the new system, use the action and the [InertialForce](https://mooseframework.inl.gov/source/kernels/InertialForce.html) kernel to define a dynamic analysis.

Newmark-Beta and HHT-alpha time integrations are supported as usual. Rayleigh damping is not yet supported, hence `eta` in the `InertialForce` kernel has to be zero. Feel free to open an issue if you need Rayleigh damping.

**Reply by xiaoruiqi** [reply]
Dear hugary1995,
  The same error message still exists.  I guess the `action` you referred to should be `Tensor Mechanics Master Action System`, the settings are listed as follows:
```
[Modules/TensorMechanics/DynamicMaster]
  [all]
    add_variables = true
    newmark_beta = 0.25
    newmark_gamma = 0.5 
    density = 7750
    new_system = true
    incremental = false
    strain = finite
  []  
[]

[Kernels]
  [inertia_x]
    type = InertialForce
    variable = disp_x
    velocity = vel_x
    acceleration = accel_x
    beta = 0.25
    gamma = 0.5
  []
  [inertia_y]
    type = InertialForce
    variable = disp_y
    velocity = vel_y
    acceleration = accel_y
    beta = 0.25
    gamma = 0.5
  []
[]
```
